app/assets/javascripts/components/Button.tsx

```diff
@@ -0,0 +1,28 @@
+import { FunctionComponent } from 'preact';
+
+const base = `rounded px-4 py-1.75 font-bold text-sm fit-content cursor-pointer`;
+
+const normal = `${base} bg-default color-text border-solid border-gray-300 border-1 \
```

Comment:
Maybe we could call these baseClass, normalClass and primaryClass?

Reply:
Sure! Fixed in 311f50d00cef43129e5fa08819e9987a3625ee87.